package.json

```diff
@@ -78,14 +76,14 @@
     "rc": "1.2.8",
     "semver": "^7.3.4",
     "sorted-array": "^2.0.4",
-    "utf-8-validate": "^5.0.4",
     "uuid": "^8.3.2",
+    "uWebSockets.js": "github:uNetworking/uWebSockets.js#v18.14.0",
     "validator": "^13.5.2",
     "winston": "^3.3.3",
     "winston-elasticsearch": "0.15.2",
     "winston-syslog": "^2.4.4",
     "winston-transport": "^4.4.0",
-    "ws": "^7.4.4",
+    "ws": "^7.4.3",
```

Comment:
Why do we still need the `ws` library?

Reply:
Argh. Merge mishap surely, I DID remove it. :expressionless:

Reply:
fixed